### PR TITLE
r-dada2: new package at 1.14

### PR DESCRIPTION
**Pull request by codeandkey**

### CODE REVIEW

File: var/spack/repos/builtin/packages/r-dada2/package.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+from spack import *
+
+
+class RDada2(RPackage):
+    """Accurate sample inference from amplicon data with single nucleotide
+       resolution"""
+
+    homepage = "http://benjjneb.github.io/dada2/"
+    url      = "https://github.com/benjjneb/dada2/archive/v1.14.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/r-dada2/package.py]: Any reason not to fetch from bioconductor?

File: var/spack/repos/builtin/packages/r-dada2/package.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    depends_on('r@3.4:', type=('build', 'run'))
+    depends_on('r-rcpp@0.12.0:', type=('build', 'run'))
+    depends_on('r-biostrings@2.42.1:', type=('build', 'run'))
+    depends_on('r-ggplot2@2.1.0:', type=('build', 'run'))
+    depends_on('r-reshape2@1.4.1:', type=('build', 'run'))
+    depends_on('r-shortread@1.32.0:', type=('build', 'run'))
+    depends_on('r-rcppparallel@4.3.0:', type=('build', 'run'))
+    depends_on('r-iranges@2.6.0:', type=('build', 'run'))
+    depends_on('r-xvector@0.16.0:', type=('build', 'run'))
+    depends_on('r-biocgenerics@0.22.0:', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/r-dada2/package.py]: Missing dependency on `gmake`

File: var/spack/repos/builtin/packages/r-dada2/package.py (hunk 1 above)
Comment (codeandkey) [reply var/spack/repos/builtin/packages/r-dada2/package.py]: Didn't actually know this one was in bioconductor; however, it looks like 1.14 is currently only available from the GitHub.

File: var/spack/repos/builtin/packages/r-dada2/package.py (hunk 2 above)
Comment (codeandkey) [reply var/spack/repos/builtin/packages/r-dada2/package.py]: Added!

File: var/spack/repos/builtin/packages/r-dada2/package.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+    depends_on('gmake')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/r-dada2/package.py]:
```suggestion
    depends_on('gmake', type='build')
```

**Pull request merged**